backend/app/utils.py

```
@@ -1,117 +1,101 @@
     def exchange_code_for_token(self, code: str):
-        """Exchange authorization code for an access token."""
-        token_url = "https://oauth2.googleapis.com/token"
         data = {
             "client_id": GOOGLE_CLIENT_ID,
             "client_secret": GOOGLE_CLIENT_SECRET,
             "code": code,
             "grant_type": "authorization_code",
             "redirect_uri": REDIRECT_URI,
         }
-        response = http_requests.post(token_url, data=data)
+        response = http_requests.post("https://oauth2.googleapis.com/token", data=data)
```

Ruff (0.8.2)

Probable use of `requests` call without timeout

(S113)